.github/workflows/release.yml

```
@@ -91,56 +91,56 @@ jobs:
       - name: Notify Success
         if: success() && github.event.inputs.dry_run != 'true'
+        env:
+          RELEASE_TAG: ${{ startsWith(github.ref, 'refs/tags/v') && github.ref_name || format('v{0}', env.NEW_VERSION) }}
+          RELEASE_VERSION: ${{ env.TAG_VERSION || env.NEW_VERSION }}
         run: |
-          echo "🎉 Successfully released snstr@${{ env.TAG_VERSION || env.NEW_VERSION }} to NPM!"
+          echo "🎉 Successfully released snstr@${{ env.RELEASE_VERSION }} to NPM!"
           echo "📦 Package: https://www.npmjs.com/package/snstr"
-          echo "🏷️ GitHub Release: https://github.com/${{ github.repository }}/releases/tag/${{ github.ref_name || format('v{0}', env.NEW_VERSION) }}" 
\ No newline at end of file
+          echo "🏷️ GitHub Release: https://github.com/${{ github.repository }}/releases/tag/${{ env.RELEASE_TAG }}"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)